sc-memory/sc-memory/src/scs/scs_parser.cpp

```diff
@@ -479,15 +479,22 @@ void Parser::ProcessTriple(
 
     if (source.IsElementType() && (connector.GetType() == ScType::ConstPermPosArc))
```

Comment:
type is still not printed(I am adding non-role to sc_node_class)
![image](https://github.com/user-attachments/assets/66bb6231-e044-4f40-b7a1-1e9310c6ba6b)